downstairs/src/region.rs

```diff
@@ -400,6 +400,30 @@ impl Region {
             assert_eq!(self.get_opened_extent(eid).number, eid);
         }
         assert_eq!(self.def.extent_count() as usize, self.extents.len());
+        use rayon::prelude::*;
+        let errors: Vec<_> = self
+            .extents
+            .par_iter()
+            .filter_map(|e| {
+                let ExtentState::Opened(extent) = e else {
+                    unreachable!("got closed extent");
+                };
+                if let Err(err) = extent.validate() {
+                    Some((extent.number, err))
+                } else {
+                    None
+                }
+            })
+            .collect();
+        if !errors.is_empty() {
+            for (number, err) in &errors {
+                warn!(
+                    self.log,
+                    "validation falied for extent {number}: {err:?}"
+                );
+            }
+            panic!("validation failed");
```

Comment:
I worry about this panic because we can't recover from it like we can the "check during repair" panic. Eventually the downstairs service will be in maintenance and the Upstairs can't do anything to fix this.